src/shared/tasks/sentryconfig/tasksentryconfig.h

```diff
@@ -0,0 +1,27 @@
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef TASKSENTRYCONFIG_H
+#define TASKSENTRYCONFIG_H
+
+#include <QByteArray>
+#include <QObject>
+
+#include "task.h"
+
+class TaskSentryConfig final : public Task {
```

Comment:
In general, it would be very useful to add a class header comment that summarizes purpose and usage. But that pattern doesn't seem to be used in this code base.

Reply:
I appreciate the callout :)